packages/config/config/devices/0x044c/myot2.json

```
@@ -0,0 +1,232 @@
+{
+	"manufacturer": "SmartDHOME",
+	"manufacturerId": "0x044c",
+	"label": "MyOT v2",
+	"description": "OpenTherm Actuator v2",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x0002"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Maximum Boiler Setpoint",
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "2",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Minimum Boiler Setpoint",
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "3",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Maximum DHW Setpoint",
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "4",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Minimum DHW Setpoint",
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "5",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Maximum Cool Setpoint",
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "6",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Minimum Cool Setpoint",
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "10",
+			"$import": "~/templates/master_template.json#base_0-255_nounit",
+			"label": "Current Error Flag",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "11",
+			"$import": "~/templates/master_template.json#base_0-255_nounit",
+			"label": "Current Error Code",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "12",
+			"$import": "~/templates/master_template.json#base_0-255_nounit",
+			"label": "Current Error Sub Code",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "20",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Pump State",
+			"readOnly": true
+		},
+		{
+			"#": "21",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Comp State",
```

```suggestion
			"label": "Compressor State",
```
or is that something else?